packages/select/src/vaadin-select.d.ts

```diff
@@ -8,7 +8,7 @@ import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.
 import { DelegateFocusMixin } from '@vaadin/field-base/src/delegate-focus-mixin.js';
 import { FieldAriaMixin } from '@vaadin/field-base/src/field-aria-mixin.js';
 import { LabelMixin } from '@vaadin/field-base/src/label-mixin.js';
-import { SlotMixin } from '@vaadin/field-base/src/slot-mixin.js';
+import { SlotMixin } from '@vaadin/component-base/src/slot-mixin.js';
```

Comment:
Please update `package.json` for the `select` package to include the `component-base` dependency.